library/Pipeline/test/Test/RecGroups.hs

```diff
@@ -37,8 +37,8 @@ pipeline =
           Right c <-
             Pipeline.toCore
               ( withJuvixExamplesPath
-                  <$> [ "positive/michelson/rec-groups/Rec-Groups.ju",
-                        "positive/michelson/rec-groups/Rec-Groups-Helper.ju"
+                  <$> [ "to-fix/rec-groups/Rec-Groups.ju",
+                        "to-fix/rec-groups/Rec-Groups-Helper.ju"
```

Comment:
Are these tests broken by some incompleteness in Juvix at the moment?  It'd be nice to have a comment for any `to-fix` tests explaining what needs to be implemented before they can be moved to `positive` or `negative`.

Reply:
That's a good point. At the moment, they are backend specific. So they fail for every backend, but they work for this bit of the pipeline. I'll think about a better folder structure to encapsulate all these nuances. Any suggestions welcome!